code/game/machinery/doors/door.dm

```diff
@@ -36,6 +36,9 @@
 	var/unres_sides = 0
 	//Multi-tile doors
 	var/width = 1
+	//Whether nonstandard door sounds (cmag laughter) are off cooldown.
+	var/soundready = TRUE
+	var/soundcooldown = 1 SECONDS
```

Comment:
(Will have to be changed elsewhere too of course)
```suggestion
	var/sound_ready = TRUE
	var/sound_cooldown = 1 SECONDS
```

Reply:
Done.